### PR TITLE
Fix bundle insertion using import-webflow-export

### PR DESCRIPTION
**Pull request by jbothma**
## Description

A meaningful description of proposed change in your own words.

## How is it tested automatically?

> Is the particular functionality/structure of the backend API this relies on tested?
> What unit, integration, GUI, end-to-end tests were added to test the functionality implemented/fixed in this PR?
> What other tests does it rely on so that the new tests are sufficient?

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally
- [ ] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out
- [ ] commit messages are meaningful

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/webflow/import.js [hunk 1]
```diff
@@ -16,4 +16,10 @@ function add_js(window, path) {
 exports.transformDOM = function(window, $) {
   // Add custom css
   add_stylesheet(window, "custom-css/mapchip.scss")
+
+  $('script[src="https://gcro.openup.org.za/js.117393d3.js"]').remove();
```
Comment (michaelglenister) [inline src/js/webflow/import.js:20]: Looks like this script was no longer used

File: src/index.html [hunk 2]
```diff
@@ -3824,4 +3824,4 @@ <h4>Indicator chart title</h4>
 }
 </style>
 
-</body></html>
+<script src="js/index.js"></script></body></html>
```
Comment (michaelglenister) [inline src/index.html:3827]: Doesn't this get added by the import script now? - The import script included index.html right?

**Review by michaelglenister — COMMENTED** [review]
I'm just not sure why index.js is referenced in index.html and the import script

File: src/index.html (hunk 2 above)
Comment (michaelglenister) [reply src/index.html:3827]: Ah, yes it does, which is why it's there

**Review by michaelglenister — APPROVED** [review]

**Comment by emre2038** [comment]
@jbothma could you resolve the conflicts please

**Pull request merged**